snap/snapcraft.yaml

```diff
@@ -27,59 +21,22 @@ apps:
     common-id: org.gnome.Fractal
     desktop: usr/share/applications/org.gnome.Fractal.desktop
     plugs:
-      - home
-      - removable-media
       - network
+      - network-bind
```

Comment:
Are you sure we need network-bind?

Reply:
This is tbh not sure of. But, there was one app I tried, it took more time to use the internet without `network-bind`. So, I simply added it here also. But, as you said, I think I should look into it.

Reply:
Please remove network-bind, it shouldn't be needed

Reply:
Done